server/src/main/resources/prometheus.yml

```
@@ -4,17 +4,17 @@ global:
+    metrics_path: '/actuator/prometheus'
+    static_configs:
+      - targets: ['host.docker.internal:8761']  # Eureka 서버의 Prometheus 메트릭 엔드포인트
 
-#  - job_name: 'eureka-client'
-#    metrics_path: '/actuator/prometheus'  # Spring Boot Actuator Prometheus 엔드포인트
-#    static_configs:
-#      - targets: ['host.docker.internal:19041', 'host.docker.internal:19042']  # Eureka 클라이언트들의 Prometheus 메트릭 엔드포인트
+  - job_name: 'grafana'
+    scrape_interval: 5s
+    static_configs:
+      - targets: [ 'grafana:3000' ]  # Eureka 서버의 Prometheus 메트릭 엔드포인트
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)